src/components/user_items_page/UserItems.js

```diff
@@ -2,28 +2,40 @@ import React, { useState, useEffect } from 'react';
 import ItemsPage from 'components/all_items_page/ItemsPage';
 import UserItemCard from './UserItemCard';
 import NoContent from 'components/UI/NoContent';
+import Error from 'components/UI/Error';
 
 const UserItems = (props) => {
     const [userItems, setUserItems] = useState(null);
-    const [error, setError] = useState(null);
+    const [errors, setError] = useState(null);
     const { apiHandler, match: { params: { uuid }} } = props;
 
+    const fetchBids = async () => {
+        const { data, errors } = await apiHandler.get(`/api/users/${uuid}/items`);
```

Comment:
That definitely sounds like a good approach! I had wanted to try and fit this refactor in, but because it would be a pretty large refactor impacting all our tests as well, would it be ok to push this to after the deadline tomorrow? We still have another story and tasks to complete